meetings/2024-01-17.md

```diff
@@ -0,0 +1,161 @@
+* MdJ: If Sarven is not regularly grooming, then it's not good to link. Unless RG you want to pick it up yourself.
+* SC: Virginia will take care access to chairs and can say more on project board e.g. STM. 
+* VB: How are they related? Right for STM, not for issues. If that can be useful we can add more recent issues or talk about in these meetings.
+* MdJ: We shouldn't link to it until someone works on it.
+* HZ: I can help if we agree there is value.
+* VB: I can look into using the board as mentioned.
+* HZ: If we agree it has value I can groom it.
+* MdJ: I don't see particular value in it. There's more value if people want to do more work on the spec to do more PR reviewing, higher priority.
+* HZ: Agree but quite a few open issues.
+* VB: Agree with MDJ. Focus on PR review, and if time/resources are available.
+* MdJ: Can use these meetings as "our board".
+* eP: I'd support it. I can join HZ on this for next week discussion.
+* HZ: Sounds good.
+* RG: This is so that anybody who comes to this can have a centralised view which might make it easier.
+* MdJ: I'd be careful to not pick up work that we might not need. I'd say we try to work on the PRs and feel that not enough clarity and other issues to work, we can dig them up.
+
+
+#### Clarify requests with N3 document in server-representation-turtle-jsonld
+URL: https://github.com/solid/specification/pull/608
+
+* MdJ: This clarifies that when there is an N3 document, it should the same as RDF document. I suggested for the purposes of N3 patch
+* eP: I want to clarify the intent of this. In LDP we talk about (Non)RDF-Sources. Here we try to avoid that distinction but here we do. With N3 it happens to use an N3 document. The way to formulate may be to consider whether SPARQL Update may be used in the future. Take some time and see if we can formulate it in a way that's special for N3 but can also work with SPARQL. Also consideration on what's mature that can go to spec.
+* MdJ: Want to make a suggestion into the spec of that wording?
+* eP: What do we think about instead of going through content types, like RDF and non-RDF sources. Then we don't have to list all patches.
+* SC: I asked for clarification in the chat. Some of that still applies. The requirement is more of a category of formats. The way it's written is not tied to a particular patch format. The only thing the spec says is patch is done this way, completely orthogonal to the formats that a server may be accepting, and if it does it needs to make available in ??? Right now the spec implies a server should be able to take an N3 document when you make a patch. Right now we're only saying RDF document but that is not entirely true. IF we want to say SPARQL is needed, that can be captured along those lines. There is nothing more generic to capture... right now we capture those two. 
```

Comment:
```suggestion
* eP: What do we think about ??? instead of going through content types, like RDF and non-RDF sources. Then we don't have to list all patches.
* SC: I asked for clarification in the chat. Some of that still applies. The requirement is more of a category of formats. The way it's written is not tied to a particular patch format. The only thing the spec says is _patch is done this way_, completely orthogonal to the formats that a server may be accepting, and if it does it needs to make available in ??? Right now the spec implies a server should be able to take an N3 document when you make a patch. Right now we're only saying RDF document, but that is not entirely true. IF we want to say SPARQL is needed, that can be captured along those lines. There is nothing more generic to capture... right now we capture those two. 
```